documents/src/pages/elements/text-field.md

```diff
@@ -48,9 +48,118 @@ p {
 
 ::
 
-`ef-text-field` is a form element for text.
+`ef-text-field` is a form element for text. This basic text input enables the user to type a small amount of text and submit it into Workspace.
 
-## Usage
+## Design Usage Rules
+
+### Content principles
+
+* Users should understand the state of the field quickly and easily
+* Placeholder text should not contain instructions. If these are needed supportive text should be provided
+* Label text can wrap onto multiple lines if needed. Avoid truncating, where possible  
+* Supportive  text should be used only where needed
+
+### Formatting
+
+Form labels should be visible and there are 3 variants:
+
+#### Stacked
+
+Stacked is the default form label location
+
+::
+
+```javascript
+::text-field::
+```
+
+```css
+label {
+  display: block;
+}
+ef-text-field {
+  width: 240px;
+}
+```
+
+```html
+<label id="name">Full Name</label>
+<ef-text-field 
+  aria-labelledby="name"
+  placeholder="Your name as shown on your passport">
+</ef-text-field>
+```
+
+::
+
+#### Left Label
+
+Use a left form label when space is vertically limited
+
+::
+
+```javascript
+::text-field::
+```
+
+```css
+ef-text-field {
+  width: 240px;
+}
+```
+
+```html
+<label id="name">Full Name</label>
+<ef-text-field 
+  aria-labelledby="name"
+  placeholder="Your name as shown on your passport">
+</ef-text-field>
+```
+
+::
+
+#### Hidden Label
+
+The form label can be hidden when needed as long as an aria-label is included
+
+::
+
+```javascript
+::text-field::
+```
+
+```css
+ef-text-field {
+  width: 240px;
+}
+```
+
+```html
+<ef-text-field 
+  aria-label="Full name"
+  placeholder="Your name as shown on your passport">
+</ef-text-field>
+```
+
+::
+
+### Copy and UX Writing
+
+See the <a href="https://lsegroup.sharepoint.com/sites/ProductDesignTeam/SitePages/Content-Design.aspx" target="_blank" rel="noreferrer">Content Design Style Guide (internal only)</a>
+
+### Touch Devices and Virtual Keyboards
+
+When interacting with `ef-text-field` using a mobile or touch device ensure that the correct virtual keyboard is activated upon user focus.
+
+### Validation and Errors
+
+Use inline validation where possible. For example, if the user typed an email address into a name field. Show the user an error message when we know the user has made an error. If it is not possible to use inline validation, then validate when the user submits the entry. For example, on a form.
+
+Inline validation is when validation messages are shown immediately after the user types in data to form fields.
+
+## Implementation
```

Comment:
It would be good to update the label styles and spacing in the other sandbox examples under implementation to reduce support queries

Reply:
Agree with Dave. @wattachai-lseg can we please update all demo blocks to have a correct style on label and text field?

Reply:
@dtanp-rft  sure thing!

Reply:
@wattachai-lseg I think this comment can be reviewed by development team. Let's find another developer to review if all live demo block use correct styles and then we can resolve this.

Reply:
@wattachai-lseg would it be beneficial to add label styles in our themes to help with this formatting? Not sure if you've thought about this already.

Reply:
That's a good idea! Once label style is stable, I will add it to the theme.

Reply:
We are using body 4 as the form level style from the '.FORM LABEL [sub]' component so that it can be applied elsewhere. 

I don't see the value-adding a dedicated style given the label isn't included in the text field, and being an atom, there will be more variances in use (this was found in last year's pattern work).

Phase 1 will address this.

Reply:
@Dave-Musson Couldn't agree more about component theme. However in this case, I believe Trem is talking about theme for the doc itself.